mode/examples/Contributed Libraries in Python/OpenCV/LiveCamTest/LiveCamTest.pyde

```diff
@@ -15,6 +17,8 @@ def setup():
 
 
 def draw():
+    global opencv
+    global video
```

Comment:
These global statements are not necessary, since you're not writing to those names.